src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/BaseExternalTaskHandler.java

```diff
@@ -39,13 +38,14 @@ public interface BaseExternalTaskHandler extends ExternalTaskHandler {
     default void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
         String topicName = externalTask.getTopicName();
         String processInstanceId = externalTask.getProcessInstanceId();
+        boolean handleTaskSucceeded = false;
 
         try {
             log.info("External task '{}' started with processInstanceId '{}'",
                      topicName, processInstanceId
             );
             handleTask(externalTask);
-            completeTask(externalTask, externalTaskService);
+            handleTaskSucceeded = true;
```

Comment:
No logic change, it just added flag, do we need this?

Reply:
there is a logic change, if the handle task succeeds then the completeTask method is called down below, otherwise it doesn't.